README.md

```diff
@@ -1,84 +1,61 @@
-# `jest-runner-tsd`
+# jest-runner-tsd
 
-A Jest runner that tests typescript typings using [tsd](https://github.com/SamVerschueren/tsd) under the hood.
+> Run your TypeScript type tests using Jest.
 
-## Install
-
-Install `jest-runner-tsd`
+[![version](https://img.shields.io/npm/v/jest-runner-tsd.svg)](https://npmjs.com/package/jest-runner-tsd)
+[![license](https://img.shields.io/github/license/jest-community/jest-runner-tsd.svg)](https://github.com/jest-community/jest-runner-tsd/blob/main/LICENSE.md)
 
-```bash
-yarn add --dev jest-runner-tsd
+Note that since `v2` the `jest-runner-tsd` is using [`tsd-lite`](https://github.com/mrazauskas/tsd-lite) instead of [`tsd`](https://github.com/SamVerschueren/tsd). The type testing logic is the same in both implementations, i.e. both of them are wrapping around the [TypeScript compiler](https://github.com/microsoft/TypeScript/wiki/Using-the-Compiler-API) to analyze types of your code.
 
-# or with NPM
+## Differences from `tsd`
 
-npm install --save-dev jest-runner-tsd
-```
+The usage of `tsd` with monorepos written in TypeScript may become [cumbersome](https://github.com/SamVerschueren/tsd/issues/32), because of checks unrelated with types. `tsd-light` is an attempt to address these and similar issues.
 
-### Adding to Jest Config
+Notable differences:
 
-Create a `jest.config.types.js` file and have the runner property set to `jest-runner-tsd` as shown below:
+- The `tsd` configuration in `package.json` is ignored. Please use `jest.config` to configure discovery of your test files and `tsconfig.json` to provide configuration for TS compiler.
```

Comment:
does this mean if you have an `index.d.ts` file in root it cannot have its own config?

Reply:
In this case TS compiler configure is used for tests. These will live in separate directory most probably. So the root `tsconfig.json` may apply or can be extend or even overridden by adding a config to that directory. I have to rewrite this paragraph. Thanks for the note.

Reply:
right, just wondering if we should accept a path somehow so people can point to a separate config file if they want... probably not needed

Reply:
Isolate a test file in a directory, drop a specific `tsconfig.json` there. Seems simple and clear. Later just add one more test file to that folder and all works.

Could be there are cases when accepting a path to config file is necessary, but I would wait to see if someone will open an issue.

It could be good idea to have `Configuration` section in the readme with more details. I will think about it.